glue/sales_val_flagging.py

```diff
@@ -572,7 +563,7 @@ def write_to_table(df, table_name, s3_warehouse_bucket_path, run_id):
         INNER JOIN default.vw_pin_sale sale
             ON sale.pin = data.pin
             AND sale.year = data.year
-        LEFT JOIN sale.flag flag
+        LEFT JOIN {args["sale_flag_table"]} flag
```

Comment:
New as of last review cycle, we are now parameterizing the tablename based on the environment! Staging environments will pull from a staging version of `sale.flag` created for the PR.